sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -778,6 +780,22 @@ private[hive] class HiveClientImpl(
   }
 
 
+
+}
+
+private[hive] object HiveClientImpl {
+  private lazy val shimDefault = IsolatedClientLoader.hiveVersion(
```

Comment:
Yes, but I am afraid we might accidentally use this default one for the interaction of Hive MetaStore in the future.